packages/core/cli.ts

```diff
@@ -141,7 +135,20 @@ yargs(process.argv.slice(2))
 					alias: "h",
 					group: "Fuzzer:",
 					default: [],
-				});
+				})
+				.array("expected_errors")
+				.option("expected_errors", {
+					describe:
+						"Expected errors can be specified as class name of the thrown " +
+						"error object or value of a thrown string. If expected errors " +
```

Comment:
Maybe: "If expected errors are defined, but none, or none of the expected ones are raised during execution, the test execution fails. Examples: -x Error ..."

Reply:
done